internal/bind/numeric_args.go

```diff
@@ -16,9 +16,21 @@ func (m NumericArgs) blockID() blockID {
 	return blockYQL
 }
 
-func (m NumericArgs) RewriteQuery(sql string, args ...interface{}) (
-	yql string, newArgs []interface{}, err error,
-) {
+func prepareArgs(args []interface{}) ([]interface{}, error) {
```

Comment:
1. `prepareArgs` is bad name, because I can't understand what do the function by the name. "Prepare" is very common word.

What about `parsePositionalParameters` or `positionalParametersToYDBParams`?

2. `newArgs` contains values with type `*params.Parameter`, in that case will be better to return `[]*params.Parameter` then `[]interface{}`